src/vellum/workflows/outputs/base.py

```diff
@@ -198,8 +200,27 @@ def __init__(self, **kwargs: Any) -> None:
         for name, value in kwargs.items():
             setattr(self, name, value)
 
-        if hasattr(self, "_outputs_post_init") and callable(self._outputs_post_init):
```

Comment:
I had been dying to get rid of this hack, and found out how. turns out any method in python has access to variables in its parent context 🤯